Source/WebCore/css/ComputedStyleExtractor.h

```diff
@@ -102,6 +102,7 @@ class ComputedStyleExtractor {
     Ref<CSSValue> getMaskShorthandValue();
     Ref<CSSValueList> getCSSPropertyValuesForGridShorthand(const StylePropertyShorthand&);
     Ref<CSSValue> fontVariantShorthandValue();
+    RefPtr<CSSValue> whiteSpacePropertyValue(const RenderStyle&);
```

Comment:
whiteSpaceShorthandValue would be a better name given the other names here (sorry only noticed this now).